docs/status.md

```diff
@@ -58,7 +58,7 @@ one of the following tracking labels.
 | concatenate              | yes           | yes          | yes            | yes             | no          |
 | constant                 | yes           | yes          | yes            | yes             | yes         |
 | convert                  | no            | yes*         | infeasible     | yes             | no          |
-| convolution              | no            | revisit      | no             | revisit         | no          |
+| convolution              | no            | yes*         | yes*           | revisit         | no          |
```

Comment:
One point here:
Currently, with client HLO not supporting unknown dimensions and StableHLO supporting it, we cannot say a `yes*` here.

Reply:
Change to revisit for both.

Reply:
Now we decide that StableHLO does not support dynamism either. So I decide to change back to yes*. Please let me know if you have other ideas.